Process.h

```diff
@@ -281,6 +281,7 @@ int Process_compare(const void* v1, const void* v2);
 void Process_delete(Object* cast);
 extern const ProcessFieldData Process_fields[LAST_PROCESSFIELD];
 #define PROCESS_MAX_PID_DIGITS 19
+#define PROCESS_MIN_UID_DIGITS 5
 #define PROCESS_MAX_UID_DIGITS 19
```

Comment:
Analog for PIDs too?

```suggestion
#define PROCESS_MIN_PID_DIGITS 7
#define PROCESS_MAX_PID_DIGITS 19
#define PROCESS_MIN_UID_DIGITS 5
#define PROCESS_MAX_UID_DIGITS 19
```

Reply:
I added a commit with a minimum number of PID digits, but setting this to 7 actually increases the size on MacOS/FreeBSD/DragonFlyBSD (which have a default of 99999). Maybe this should be 5 as well?

Reply:
If its determined dynamically anyway keeping it at 5 won't hurt.

Reply:
Ok! I updated the PR with `PROCESS_MIN_PID_DIGITS` set to `5`.